ui/src/app/components/cromwell-configuration-panel.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from 'react';
+import { useEffect } from 'react';
+import * as fp from 'lodash/fp';
+
+import { AppType } from 'generated/fetch';
+
+import { Button } from 'app/components/buttons';
+import { FlexColumn, FlexRow } from 'app/components/flex';
+import { WarningMessage } from 'app/components/messages';
+import { styles } from 'app/components/runtime-configuration-panel/styles';
+import { Spinner } from 'app/components/spinners';
+import { appsApi } from 'app/services/swagger-fetch-clients';
+import {
+  withCdrVersions,
+  withCurrentWorkspace,
+  withUserProfile,
+} from 'app/utils';
+import { findMachineByName } from 'app/utils/machines';
+import { runtimeStore, useStore } from 'app/utils/stores';
+
+import { defaultCromwellConfig } from './apps-panel/utils';
```

Comment:
Should this file be located somewhere else? I am wondering if it is worth having a configuration panel directory.

Reply:
I'm fine with either. It's easy to change this later as well.

Reply:
`components/runtime-configuration-panel/` and `components/apps-panel/`both exist btw.

Reply:
Are you suggesting that we should put it in one of those directories, or are you suggesting that there is convention for making directories?

Reply:
I have not thought about it enough to make a specific suggestion - and now I see that obviously you know about them because you're updating them here.  never mind :)